logging.conf

```diff
@@ -8,7 +8,7 @@ keys=console_handler
 keys=default_formatter
 
 [logger_root]
-level=INFO
+level=NOTSET
```

Comment:
there are going to be lots of messages, in this case.

Reply:
I set it to "DEBUG", will be visible into the next commit. I propose we keep it this way at least for a while, it will be easier for us to do post-mortem analysis.